@types/index.d.ts

```diff
@@ -419,6 +428,26 @@ declare module "opentok-react-native" {
      * Sent when the publisher stops streaming.
      */
     streamDestroyed?: CallbackWithParam<StreamDestroyedEvent, any>;
+
+    /**
+     * Sent when the publisher stops sending video because of [publisher audio fallback](https://tokbox.com/developer/guides/audio-fallback).
```

Comment:
"Sent when the publisher stops sending video because of" seems to be incomplete. Did you mean the link place holder as next word?